src/client/pages/Checkout/CheckoutDetails/CheckoutDetails.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
-import { useTextKeys } from 'utils/textKeys'
-import { useQuoteCartIdFromUrl } from 'utils/hooks/useQuoteCartIdFromUrl'
+
 import { useCurrentLocale } from 'l10n/useCurrentLocale'
 import { useQuoteCartData } from 'utils/hooks/useQuoteCartData'
+import { useQuoteCartIdFromUrl } from 'utils/hooks/useQuoteCartIdFromUrl'
+import { useTextKeys } from 'utils/textKeys'
```

Comment:
we don't need to import these ones right ? 
import { useQuoteCartIdFromUrl } from 'utils/hooks/useQuoteCartIdFromUrl'
import { useTextKeys } from 'utils/textKeys'

Reply:
they are used within the file